app/src/main/java/com/orange/ods/app/ui/components/buttons/icons/ButtonsIconToggleGroup.kt

```diff
@@ -42,7 +42,7 @@ import com.orange.ods.compose.theme.OdsDisplaySurface
 fun ButtonsIconToggleGroup(customizationState: ButtonIconCustomizationState) {
     val iconToggleButtons =
         LocalRecipes.current.distinctBy { it.iconResId }.filter { it.iconResId != null }.take(ButtonIconCustomizationState.MaxToggleCount).map { recipe ->
-            OdsIconToggleButtonsRowItem(painterResource(id = recipe.iconResId!!), recipe.title)
+            OdsIconToggleButtonsRowItem(painterResource(id = recipe.iconResId!!), recipe.title, customizationState.enabled.value)
```

Comment:
I think the documentation should be updated with this parameter and the code implementation should also be changed to display the enabled value.